.github/workflows/push-artifacts.yaml

```
@@ -62,17 +65,15 @@ jobs:
 
-      - name: Test operator
-        working-directory: operator
+      - name: Run checks
         run: make check
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v4
 
       - name: Build and push operator image
-        working-directory: operator
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:41: Double quote to prevent globbing and word splitting

(shellcheck)